packages/gatsby-plugin-gatsby-cloud/src/__tests__/gatsby-browser.js

```diff
@@ -210,22 +212,9 @@ describe(`Preview status indicator`, () => {
   //   })
   // })
 
-  /**
-   * SKIPPED TEST NOTE
-   * 1. The previous tests were written withe the assumption that the tooltips were
-   * displayed but not just not visible. Since logic was added that truly made the
-   * tooltips dissapear the current tests failed. In an effort to fix the these we
-   * ran into multiple issues concerning state and events that will take some refactoring to fix.
-   *
-   * 2. These tests only concern the hiding and showing the tooltip in certain cases
-   * so should affect coverage adversely
-   *
-   * 3. A PR to fix these test and other issues will be added when we refactor the plugin
-   */
-
```

Comment:
we're still skipping a bunch of tests, is this comment not relevant anymore?

Reply:
its relevant, ill add a note to the skipped tests.